coriolis/api-refs/api_samples/migration/openstack-migration-list-detail-resp.json

```diff
@@ -0,0 +1,344 @@
+{
+  "migrations": [
+    {
+      "created_at": "2019-07-24T13:03:45.000000",
+      "updated_at": "2019-07-24T13:09:16.000000",
+      "deleted_at": null,
+      "deleted": "0",
+      "base_id": "e4f432bd-393d-41c5-ab78-04b68b70d4ee",
+      "user_id": "7cf9b4745d514517b3eae440438136bc",
+      "project_id": "ffc16db2011e41f28152897c279b6bcc",
+      "destination_environment": {
+        "storage_mappings": {
+          "lvmdriver-1": "lvm-1"
+        },
+        "keypair_name": "new-key",
+        "delete_disks_on_vm_termination": false,
+        "migr_image_map": {
+            "linux": "ubuntu-xenial-img",
+            "windows": "63d8f1a4-3192-4edc-b113-0d099b4bc458"
+        },
+        "migr_network": "provider-net",
+        "port_reuse_policy": "keep_mac",
+        "set_dhcp": true
+      },
+      "type": "migration",
+      "instances": [
+        "ubuntu-xenial"
+      ],
+      "reservation_id": null,
+      "notes": "",
+      "origin_endpoint_id": "e19100c7-cc6e-4890-9ce6-4d638b1b5ede",
+      "destination_endpoint_id": "73f93b88-e0b2-4e1e-8f7d-345ae3000e3b",
+      "transfer_result": null,
+      "network_map": {
+        "private": "provider-net"
+      },
+      "storage_mappings": {
+
+      },
+      "source_environment": {
+
+      },
+      "id": "e4f432bd-393d-41c5-ab78-04b68b70d4ee",
+      "replica_id": null,
+      "status": "ERROR",
+      "tasks": [
+        {
+          "created_at": "2019-07-24T13:03:45.000000",
+          "updated_at": "2019-07-24T13:03:48.000000",
+          "deleted_at": null,
+          "deleted": "0",
+          "id": "b18f11c0-3f06-453f-81dd-433b8fb02b69",
+          "execution_id": "0121ccbb-ad06-4672-92ef-e908be0096cc",
+          "instance": "ubuntu-xenial",
+          "host": "coriolis-worker",
+          "process_id": 74,
+          "status": "COMPLETED",
+          "task_type": "VALIDATE_MIGRATION_SOURCE_INPUTS",
+          "exception_details": null,
+          "depends_on": null,
+          "on_error": false,
+          "events": [],
+          "progress_updates": [
+            null
+          ]
+        },
+        {
+          "created_at": "2019-07-24T13:03:45.000000",
+          "updated_at": "2019-07-24T13:04:05.000000",
+          "deleted_at": null,
+          "deleted": "0",
+          "id": "79bd38dd-49c9-46c9-8726-0a854fe5ea3f",
+          "execution_id": "0121ccbb-ad06-4672-92ef-e908be0096cc",
+          "instance": "ubuntu-xenial",
+          "host": "coriolis-worker",
+          "process_id": 79,
+          "status": "COMPLETED",
+          "task_type": "VALIDATE_MIGRATION_DESTINATION_INPUTS",
+          "exception_details": null,
+          "depends_on": [
+            "b18f11c0-3f06-453f-81dd-433b8fb02b69"
+          ],
+          "on_error": false,
+          "events": [],
+          "progress_updates": [
+            null,
```

Comment:
Although I appreciate your ingenuity in loading the JSON and replacing progress updates with `null`, please do it properly without leaving any "evidence" ;)

Reply:
I didn't put that null there, I guess it's there because the migration was completed?